python/lsst/meas/base/forcedPhotCoadd.py

```diff
@@ -52,8 +53,8 @@ def setDefaults(self):
 
     def validate(self):
         ForcedPhotImageTask.ConfigClass.validate(self)
-        if (self.measurement.doReplaceWithNoise and self.footprintDatasetName is not None
-                and self.references.removePatchOverlaps):
+        if (self.measurement.doReplaceWithNoise and self.footprintDatasetName is not None and
+                self.references.removePatchOverlaps):
```

Comment:
It was W503. Weirdly, `pycodestyle` implies that W503 is disabled by default (this was changed in the past year). I am using the current version so it's almost like my `flake8` is forcing it to be enabled. I've added it to the ignore list explicitly. I wasn't able to install the newest flake8 as an eups package because of some problem with `setuptools` version numbering.